app.js

```diff
@@ -3,6 +3,8 @@ const cors = require('cors');
 
 require('dotenv').config();
 
+const clams = require('./routes/clams');
```

Comment:
let's call this `clamsRouter` so we can distinguish what we're importing from `clams.js` better